From 2881eafd48095bae544cbb857c02b6166a21e80e Mon Sep 17 00:00:00 2001
From: Venkataramana Kotakonda <venkataramana.kotakonda@intel.com>
Date: Fri, 11 Dec 2015 16:06:57 +0530
Subject: [PATCH 3/8] USBC/PHY: Disable auto soft and hard resets in fusb

The soft and hard reset should be handled in policy engine. Auto soft
and hard reset in hardware (fusb) will trigger unnecessary resets.

Change-Id: I2d1f5d7601330cb8658057e53599044e70d0e222
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-12097
Signed-off-by: Venkataramana Kotakonda <venkataramana.kotakonda@intel.com>
Reviewed-on: https://android.intel.com:443/447410
---
 drivers/usb/typec/phy-fusb300.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/usb/typec/phy-fusb300.c b/drivers/usb/typec/phy-fusb300.c
index 6fc9be6..aaa3cbd 100644
--- a/drivers/usb/typec/phy-fusb300.c
+++ b/drivers/usb/typec/phy-fusb300.c
@@ -659,10 +659,7 @@ static inline int fusb302_configure_pd(struct fusb300_chip *chip)
 	unsigned int val;
 
 	val = FUSB302_CONTROL3_AUTO_RETRY |
-		(FUSB302_CONTROL3_RETRY3 << FUSB302_CONTROL3_N_RETRY_SHIFT) |
-		FUSB302_CONTROL3_AUTO_SOFT_RST |
-		FUSB302_CONTROL3_AUTO_HARD_RST;
-
+		(FUSB302_CONTROL3_RETRY3 << FUSB302_CONTROL3_N_RETRY_SHIFT);
 	regmap_write(chip->map, FUSB302_CONTROL3_REG, val);
 	return 0;
 }
-- 
1.9.1

